lua/entities/base_wire_entity.lua

```diff
@@ -225,21 +226,23 @@ if CLIENT then
 		ent:DrawWorldTip()
 	end)
 
+	local playerWasLookingAtMe = false
+
 	-- Custom better version of this base_gmodentity function
 	function ENT:BeingLookedAtByLocalPlayer()
-		local trbool = BaseClass.BeingLookedAtByLocalPlayer(self)
+		local trbool = beingLookedAtByLocalPlayer(self)
 
-		if self.playerWasLookingAtMe ~= trbool then
-			net.Start( "wire_overlay_request" )
+		if playerWasLookingAtMe ~= trbool then
```

Comment:
Need to replace these?

Reply:
It's another __index dip out of the Lua realm so I feel like it could be removed and replaced with a local variable to save a little more performance (this attribute is only used by this particular clientside function)

Reply:
Never mind, I misread the source the first time around and thought `playerWasLookingAtMe` was undefined.